### PR TITLE
Feature/update credits

### PR DESCRIPTION
**Pull request by ambid17**
also added unusualcrow's "urgency" soundtrack

### CODE REVIEW

File: BunnyHopMaster/Assets/Scenes/Credits.unity [hunk 1]
```diff
@@ -882,19 +886,52 @@ MonoBehaviour:
...
+    Molly
+    (VultureSkins)
+
+
+    CONCEPT ARTISTS
+
+    Marla Menendez Posada (Glossywoo)
+
+    Shervin
+    Chung ( Solstice_oo)
```
Comment (0xLeif) [inline BunnyHopMaster/Assets/Scenes/Credits.unity]: `(Solstice_oo)`

File: BunnyHopMaster/Assets/Scenes/Credits.unity [hunk 2]
```diff
@@ -882,19 +886,52 @@ MonoBehaviour:
...
+
+
+    UI DESIGNERS:
+
+    Daniel Lin ( Chum)
```
Comment (0xLeif) [inline BunnyHopMaster/Assets/Scenes/Credits.unity]: `(Chum)`

**Comment by 0xLeif** [comment]
When/ Why are the tests failing? :(

**Review by 0xLeif — CHANGES_REQUESTED** [review]
Tiny changes

File: BunnyHopMaster/Assets/Scenes/Credits.unity (hunk 2 above)
Comment (ambid17) [reply BunnyHopMaster/Assets/Scenes/Credits.unity]: ope ty

**Review by Vextin — APPROVED** [review]

**Review by 0xLeif — APPROVED** [review]

**Pull request merged**